synchronizer/common/log_helper.go

```diff
@@ -0,0 +1,42 @@
+package common
```

Comment:
Let's talk about that. Are not a check is to create the log text when comparing 2 binaries data to avoid put the full contents, to log just the difference